### PR TITLE
Upgrade undertow-standalone-jaspi to EE10 and WFLY 27.0.1

### PR DESCRIPTION
**Pull request by cam-rod**

### CODE REVIEW

File: undertow-standalone-jaspi/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+### Run the Demo App
+
+The demo app configures Elytron using [the programmatic approach](https://docs.wildfly.org/27/WildFly_Elytron_Security.html#the-programmatic-approach), so a single command will configure Elytron, deploy the application, and launch it. To do so, run the following command in a terminal:
+```shell
+mvn exec:java
+```
+
+### Access the Application
+
+The application will be running at `http://localhost:28080/helloworld`. To see and manipulate the
```
Comment (fjuma) [inline undertow-standalone-jaspi/README.md:42]: s/28080/8080

Comment (fjuma) [reply undertow-standalone-jaspi/README.md:42]: Ah, actually, I see this is what was used in the blog post. Ignore this.

File: undertow-standalone-jaspi/pom.xml [hunk 2]
```diff
@@ -15,29 +15,40 @@
 
     <groupId>org.wildfly.security.examples</groupId>
     <artifactId>undertow-standalone-jaspi</artifactId>
-    <version>2.0.0.Alpha1-SNAPSHOT</version>
+    <version>3.0.0.Alpha1-SNAPSHOT</version>
```
Comment (fjuma) [inline undertow-standalone-jaspi/pom.xml:18]: The main branch is on 2.0.0.Alpha1

Comment (cam-rod) [reply undertow-standalone-jaspi/pom.xml:18]: Ah, I wasn't sure if those numbers should be upgraded based on breaking quickstart changes or not. Will revert

Comment (fjuma) [reply undertow-standalone-jaspi/pom.xml]: The 1.x branch contains the original versions of the examples. We bumped the main branch to 2.x, where everything is being migrated to EE 10.

**Review by fjuma — APPROVED** [review]
Thanks @cam-rod!

**Pull request merged**